dist/index .html

```diff
@@ -0,0 +1,359 @@
+            ><span class="activity"
+              >Ryan had been leading open-source projects at the Mozilla
+              Foundation such as the open source move-ment.</span
+            ></span
+          >
+        </li>
+      </ul>
+    </section>
+    <section class="about-page hidden">
+      <div class="logo">
+        <h2>Global Web Designers Summit 2021 Logo</h2>
+        <p class="mobile">
+          The logo of G.W.D Summit 2021 was decided through the logo competition
+          from 8. June to 7.July
+        </p>
+        <p class="Desktop">
+          The logo of G.W.D Summit 2021 was decided through the logo competition
+          from 8. June to 7.July.<br />After online voting and Selection
+          Committee's judging process, the logo from Naresh Agrawal won the
+          prize<br />After consultation with Mr.Agrawal we optimized the
+          prize-winning-logo for this year's summit as following:
+        </p>
+        <div class="LogoImageContainer">
+          <img
+            src="img/poth-version-img/large.png"
+            class="logoImage"
+            alt="logo image"
+          />
+        </div>
+      </div>
+      <div class="pastSummit">
+        <h2>See the past G.W.D Summits</h2>
+        <p>
+          Take a look at the last two G.W.D Global Summits which took place in
+          Buenos Aires in Warsaw
+        </p>
+        <div class="PastContainer">
+          <div class="pastImg first">
+            <h2>
+              <span>2013</span><br /><span
+                >G.W.D Summit 2013 in Buenos Aires</span
+              >
+            </h2>
+          </div>
+          <div class="pastImg second">
+            <h2>
+              <span>2011</span><br /><span>G.W.D Summit 2011 in Warsaw</span>
+            </h2>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="Partner hidden">
+      <h2>Partner</h2>
+      <div class="PartnerContainer">
+        <img
+          src="img/poth-version-img/firstParntner.png"
+          class="partImage first"
+          alt="first partner"
+        />
+        <img
+          src="img/poth-version-img/secondParntner.png"
+          class="partImage second"
+          alt="second partner"
+        />
+        <img
+          src="img/poth-version-img/thirdParntner_.png"
+          class="partImage third"
+          alt="third partner"
+        />
+        <img
+          src="img/poth-version-img/thirdParntner.png"
+          class="partImage fourth"
+          alt="fourth partner"
+        />
+        <img
+          src="img/poth-version-img/fourthParntner.png"
+          class="partImage fifth"
+          alt="fifith partner"
+        />
+      </div>
+      <div class="finalLogo">
+        <img
+          src="img/poth-version-img/FinalLogo.png"
+          class="finalLogoImage"
+          alt="final logo"
+        />
+        <p>2015 Creative Commons Korea Some Rights Reserved</p>
```

Comment:
### original footer
![original-footer](https://user-images.githubusercontent.com/80149314/142507253-5f7b732f-3660-4867-af9f-dd00714eada9.jpg)
 Take note of the space between the logo and the text to the right, the two being centered. Also don't forget to give some margin. Please make the requested changes